### DISCUSSION TITLE
Why most of the Bootstrap 5 classes not working as per the documentation ?

### DISCUSSION THREAD

**Discussion by Sumantbag1992** [twbs/bootstrap#34662] [General]
Bootstrap 5 classes like bg-color, text-color are not working as they used to in Bootstrap 5, if there is any updated documentation for the same issues, please provide it.
![image](https://user-images.githubusercontent.com/86064843/128060554-4a8efc88-15f5-49b8-aa93-0743ff4aa89c.png)
it's coming like this :
![image](https://user-images.githubusercontent.com/86064843/128060599-3f62d291-f9c5-4e87-a450-1943f8316823.png)

**Comment by mdo** [comment]
Your CSS is likely not being referenced correctly. Double check paths :).

**Reply by Sumantbag1992** [reply]
link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.4.1/css/bootstrap.min.css"

**Reply by XhmikosR** [reply]
That's v3, but you are viewing the v5 docs.

**Reply by Sumantbag1992** [reply]
it was given in the v5 docs...

**Reply by Hiws** [reply]
> it was given in the v5 docs...

You should probably mention where so it can be fixed.

**Reply by XhmikosR** [reply]
It is mentioned clearly in the docs https://getbootstrap.com/docs/5.1/getting-started/introduction/
